.github/actions/release-connect/action.yml

```diff
@@ -82,6 +82,7 @@ runs:
       shell: bash
       env:
         NODE_ENV: ${{ inputs.nodeEnv }}
+        ASSET_PREFIX: /${{ inputs.serverPath }}
```

Comment:
this is going to affect production releases, so lets double check

Reply:
Yes.  That is a good idea, I am double checking potential places where it could fail.

Reply:
Yes, I double checked the actions and it should work. We use the same action for dev environment and production so it is very close. In production the serverPath will be:

* The version relase => `serverPath: 9.4.4-beta.1`
* The static conenect v9 release => `serverPath: 9`